tests/README.md

```diff
@@ -0,0 +1,63 @@
+# Tests
+
```

Comment:
Add a sentence here to describe what the testing situation is for `clasp`. Something like:

Many of `clasp`'s CLI commands have unit tests that run the `clasp` command and asserts the std output and std error code is as expected.

Reply:
👍